src/Cake.Common/Tools/OctopusDeploy/OctopusDeployPusher.cs

```diff
@@ -0,0 +1,82 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@agc93 there are some trailing whitespaces in this file.  Can you remove?

![image](https://cloud.githubusercontent.com/assets/1271146/16129711/cf46c286-33fd-11e6-8cab-619a092b08f1.png)